### ISSUE TITLE
Avg Daily Cases Map | Voice Over | State values need description

### ISSUE THREAD

**Issue by LaurieSReynolds** [COVID19Tracking/website#1691]
Navigating the states using voice over, the description for the value is missing.

Recommendation

Code the states as a hidden table with the column heading "Average daily cases per 100k people in the last seven days"
